man/sd_bus_reply_method_return.xml

```diff
@@ -52,6 +59,9 @@
     <citerefentry><refentrytitle>sd_bus_message_append</refentrytitle><manvolnum>3</manvolnum></citerefentry>.
     If no reply is expected to <parameter>call</parameter>, this function succeeds without sending a
     reply.</para>
+
+    <para><function>sd_bus_reply()</function> takes a complete bus return message object and sends it as a
```

Comment:
"return message object" is not very clear... Maybe "a complete bus message object that is a reply message" ? Actually, it'd be good to clarify what the expectations about the message are. I assume we fill in the destination address, but are there other requirements?

Reply:
AFAIK, there aren't any other explicit requirements, just a "finishes" D-Bus message with all the types filled in and all containers closed.

Reply:
I can add that it should have been created with `sd_bus_new_method_return` or `sd_bus_new_method_error`.